ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -778,7 +797,6 @@ class RGBDecomp(object):
         arcpy.env.workspace = indir
         indirbase = os.path.basename(indir)
         # Create a scratch directory for intermediate files
-        scratch = arcpy.CreateFolder_management(indir, "temp")
```

Comment:
Does this automatically happen in the os.path.join function? If so, that's fine. But this directory needs to be created one way or another. 
Ah, I see, it was removed because it was set to be a variable, but I changed the scratchpath to not use this variable. If scratchpath doesn't generate a directory, this line should be added back in, but doesn't need to be a variable.